portal-web/test/functional/com/liferay/portalweb/paths/mypaths/MyPath.path

```diff
@@ -0,0 +1,41 @@
+<html>
+<head>
+<title>MyPath</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">MyPath</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>SIMULATION_MENU</td>
+	<td>//div[contains(@class,'control-menu')]//button[@data-qa-id='simulation']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CUSTOM_SIZE_BUTTON</td>
+	<td>//div[contains(@class,'row size-selector')]//button[child::span[text()='Custom']]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CUSTOM_SIZE_HEIGHT</td>
+	<td>//div[contains(@id,'customSizeSelector')]//input[contains(@id,'height')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CUSTOM_SIZE_WIDTH</td>
+	<td>//div[contains(@id,'customSizeSelector')]//input[contains(@id,'width')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CUSTOM_SIZE_APPLY_BUTTON</td>
+	<td>//div[contains(@id,'customSizeSelector')]//button[@aria-label='Apply Custom Size']</td>
```

Comment:
Hey, I think you could also try using the XPath like this:
`//button[@aria-label='Apply Custom Size']`
What do you think? 
Thanks!

Reply:
Okay, I changed it. Thanks!